frontend/packages/kubevirt-plugin/src/components/vm-disks/disk-summary.tsx

```diff
@@ -6,56 +6,57 @@ import {
   getPVCSourceByDisk,
 } from '../../selectors/vm/selectors';
 import { WINTOOLS_CONTAINER_NAMES } from '../modals/cdrom-vm-modal/constants';
-import { VMKind } from '../../types';
-import { V1Disk } from '../../types/vm/disk/V1Disk';
+import { VMKind, BootableDeviceType } from '../../types';
+import { deviceLabel } from '../boot-order/constants';
 
 const DiskSummaryRow: React.FC<DiskSummaryRowProps> = ({ title, value }) => (
```

Comment:
can we pass some generic id prop here, we can pass the same value as title, but IMHO it does not make sense to generate id out of value. We have `data-` attributes for that

Reply:
After the refactor :-) , the signature of this component stay untouched.